comparison_helpers.py

```diff
@@ -157,9 +160,9 @@ def summarize_results(results, labels_raw):
 def convert_to_excel(results_dict, summary_df, out_file):
     writer = pd.ExcelWriter(out_file, engine="xlsxwriter")
     summary_df.to_excel(writer, sheet_name="Overall Summary")
-    for file_name, results in results_dict.items():
+    for i, (file_name, results) in enumerate(results_dict.items()):
         df = pd.DataFrame.from_records(
             [r["original_label"] for r in results["missing"]], index=None
         )
-        df.to_excel(writer, sheet_name=file_name[:30], index=None)
+        df.to_excel(writer, sheet_name=f"{i}_{file_name[:26]}", index=None)
```

Comment:
What's the logic for the 26 vs. 30?

Reply:
sheet name has 30 character limit, and i can sometimes go up to the thousands depending on num of docs